packages/@okta/vuepress-site/docs/reference/api/sessions/index.md

```diff
@@ -25,47 +25,42 @@ Okta provides a very rich [Authentication API](/docs/reference/api/authn/) to va
 - [Retrieving a session cookie by visiting a session redirect link](/docs/guides/session-cookie/#retrieving-a-session-cookie-by-visiting-a-session-redirect-link)
 - [Retrieving a session cookie by visiting an application embed link](/docs/guides/session-cookie/#retrieving-a-session-cookie-by-visiting-an-application-embed-link)
 
-> **Session Tokens** are secrets and should be protected at rest as well as during transit. A session token for a user is equivalent to having the user's actual credentials.
+> **Session tokens** are secrets and should be protected at rest and during transit. A session token for a user is equivalent to having the user's actual credentials.
 
-## Getting Started
+## Get started
 
 Explore the Sessions API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/9f729bbb060e6bac09fa)
 
-## Session Operations
-
-### Create Session with Session Token
+## Session operations
 
+### Create session with a session token
```

Comment:
session would be Session - and throughout this doc. We capitalize objects, for example, the Session object. 

see me if you have questions about this one.